aries-site/src/examples/templates/page-layouts/components/PageContainer.js

```diff
@@ -54,9 +54,10 @@ export const PageContainer = ({ kind = 'wide', ...rest }) => {
     <PageContainerContext.Provider value={{ pad }}>
       <Box
         alignSelf={pageContainer[kind].align}
-        fill
+        fill="horizontal"
+        flex
         gap={pageContainer.gap[size]}
-        margin="auto"
+        margin="0 auto" // only horizontally center
```

Comment:
good catch

from a syntax standpoint, do we do this elsewhere? Or do we tend to follow the 
```suggestion
        margin={{ horizontal: 'auto' }}
```